### ISSUE TITLE
Request for info-help with LE errors on rebuilt server

### ISSUE THREAD

**Issue by EdNett** [omega8cc/boa#1059]
Hi,

I LOVE the LE feature, which worked great for us. Then this server didn't make the upgrade from wheezy to jessie, so I started fresh with Jessie, re-installed BOA (on the same machine with the same server name and IP address) and then installed the sites new in Octopus, then restored backups (file and db) into the new sites.  I haven't been able to get the LE certificates to work since then ( a few months now). I have very informative error messages - (not authorized messages) here:

https://gist.github.com/EdNett/4c990c779c2d11c25014b8441242ec21

The really informative error messages start at line 99 of the GIST:

[hosting_le] ACME Output: # # !! WARNING !! No main config file found, using default config! # Processing domain.com with alternative names: www.domain.com  + Signing domains...  + Creating new directory /data/disk/fast1/tools/le/certs/domain.com ...  + Generating private key...  + Generating signing request...  + Requesting challenge for domain.com...   + ERROR: An error occurred while sending post-request to https://acme-v01.api.letsencrypt.org/acme/new-authz (Status 403)  Details: {   "type": "urn:acme:error:unauthorized",   "detail": "No registration exists matching provided key",   "status": 403 }

It appears that LE has a record of the former certificates that will never match the new ones? Is that the problem. I can't have hit any rate limits - there aren't that many sites... The hostmaster site doesn't work either - and it has the same exact name as before.
Do I need to revoke the certificates that existed 2-3 months ago with LetsEncrypt directly? in order to get new certificates to be re-issued?  Is there something that I can do within BOA to try to get this to work again? What do you suggest?

Thank you.

Ed

BOA info more is here:
https://gist.github.com/EdNett/a1bbcb1c74542ae38e54e448214e5bf4

**Comment by omega8cc** [comment]
Try these steps:

1.`$ touch /data/disk/fast1/tools/le/.ctrl/ssl-demo-mode.pid`
2. Add or change some alias in **every** site with previously enabled LE SSL
3. Wait until all Verify task will run with green result
4. `$ rm -f /data/disk/fast1/tools/le/.ctrl/ssl-demo-mode.pid`
5. Re-verify all sites with SSL enabled

These steps should re-register your system at LE with new keys and force new certs.

**Comment by EdNett** [comment]
Hello,

That did not work for me, and I tried it many different ways, including running barracuda up-stable followed by octopus up-stable fast1 force in the middle. I looked for error messages, but they were the exact same error messages - only those.

Finally I had to delete everything (every file) related to LE on the server - so I 
rm -R  /data/disk/fast1/tools/le/*
rm -R /data/disk/fast1/config/ssl.d/*
rm -R /data/disk/fast1/config/server_master/ssl.d/*

and then I made sure no sites had encryption enabled, and I ran barracuda up-stable and octopus up-stable fast1 force

Then I enabled SSL for each site and verified, and it now works.
